dolphinscheduler-ui/src/sass/common/index.scss

```diff
@@ -80,7 +80,6 @@ body{
       //background: #f8fbfe;
     }
     .conditions-model {
```

Comment:
This will conflict with the pr of https://github.com/apache/dolphinscheduler/pull/6911/files, which requires local rebase.

Reply:
done.